### PR TITLE
Replace the GOV.UK blogs promo with the tax-free savings promo

### PR DESCRIPTION
**Pull request by issyl0**
This needs _deploying_ on Wednesday 4th March.

### CODE REVIEW

**Comment by benilovj** [comment]
I think we need a different image:

* the others promo are all JPGs
* the other promo images are 608 x 405 (so that they look good on an iPad)

**Comment by bradleywright** [comment]
The JPG thing is mostly because we can compress JPGs down a lot further than PNGs.

File: app/views/root/index.html.erb [hunk 1]
```diff
@@ -162,10 +162,10 @@
             </div>
             <div class="promo-image">
               <div class="promo-content">
-                <a href="https://www.blog.gov.uk"><%= image_tag 'homepage/govuk-blogs.jpg', alt: 'GOV.UK blogs' %></a>
-                <h3>GOV.UK blogs</h3>
+                <a href="/taxfreesavings"><%= image_tag 'homepage/tax-free-savings.png', alt: 'Tax-free savings' %></a>
+                <h3>Tax-free savings</h3>
                 <p>
-                  <a href="https://www.blog.gov.uk">Search the list</a> of GOV.UK blogs, and find one to match your interest.
+                  More people will be able to get tax-free interest on their savings from 6 April - <a href="/taxfreesavings">find out if you'll qualify</a>.
```
Comment (benilovj) [inline app/views/root/index.html.erb]: pedantic nitpick - would be good to use [smart quotes](http://smartquotesforsmartpeople.com/) for `you'll`

**Comment by bradleywright** [comment]
Actually looking at the image it might be better as a GIF (it's vector art, not a photo). PNG would also be suitable - worth checking the relative file size of both.

**Comment by edds** [comment]
If we are loosing the last reference to `homepage/govuk-blogs.jpg` can we also remove the old asset as part of this please.

File: app/views/root/index.html.erb (hunk 1 above)
Comment (issyl0) [inline app/views/root/index.html.erb]: Yup - amended. And there was me getting pedantic about the hyphen in the text to insert instead of a proper dash...

**Comment by issyl0** [comment]
@edds: Done.

**Comment by benilovj** [comment]
closing this until we have a bigger image

**Comment by issyl0** [comment]
Closing this until we get a better image...

**Pull request closed without merging**